js/jquery.fileupload-video.js

```diff
@@ -97,8 +98,11 @@
                 return data;
             },
 
-            // Sets the video element as a property of the file object:
+            // Copy the properties set on the data object to the file object
```

Comment:
Also, as a side item, it's not clear why this isn't done in the `loadVideo` function.